### DISCUSSION TITLE
Trying to verify student mail for github-copilot

### DISCUSSION THREAD

**Discussion by Jakob2205** [community/community#70441] [Copilot Conversations]
### Select Topic Area

General

### Body

First of all sorry if this is the wrong forum for posting this, but i saw through the different forum names and this seemed the most appropriate. 

I already have a Github account with my primary mail, and i have tried to add my student-mail to it for the github-copilot student package. But whenever i press send verification link, but i don't receive the mail.  Even though it says that it has sent a verification mail to the relevant e-mail i am trying to use. 

Have any of you experienced similar issues?

**Comment by tsantef** [comment]
This could be from the student mail being configured to block 3rd party senders outside of the school.
